frontend/src/scenes/trends/trendsLogic.ts

```diff
@@ -316,10 +315,7 @@ export const trendsLogic = kea<
     }),
 
     selectors: () => ({
-        filtersLoading: [
-            () => [teamLogic.selectors.currentTeamLoading],
-            (currentTeamLoading: any) => currentTeamLoading,
-        ],
+        filtersLoading: [() => [eventDefinitionsLogic.selectors.loaded], (loaded: boolean): boolean => !loaded],
```

Comment:
Should this also depend on the property definitions loading?

Reply:
Then again, *maybe* not? Making properties load slow does open the interface, just the dropdown for properties won't contain the full list...

<img width="890" alt="Screenshot 2021-04-23 at 15 07 55" src="https://user-images.githubusercontent.com/53387/115875714-caf9cf80-a445-11eb-8a83-3a7e333c1e56.png">

Reply:
I added it to the list in any case.

Reply:
Great point! I think we can show a local loading indicator in the properties list. Though when we add server-side querying it won't even matter anymore.